deps/chakrashim/src/v8chakra.h

```diff
@@ -77,6 +77,66 @@ class ExternalData {
   }
 };
 
+struct SetterGetterInterceptor
+{
+  NamedPropertyGetterCallback namedPropertyGetter;
+  NamedPropertySetterCallback namedPropertySetter;
+  NamedPropertyQueryCallback namedPropertyQuery;
+  NamedPropertyDeleterCallback namedPropertyDeleter;
+  NamedPropertyEnumeratorCallback namedPropertyEnumerator;
+  NamedPropertyDefinerCallback namedPropertyDefiner;
+  NamedPropertyDescriptorCallback namedPropertyDescriptor;
+  Persistent<Value> namedPropertyInterceptorData;
+  IndexedPropertyGetterCallback indexedPropertyGetter;
+  IndexedPropertySetterCallback indexedPropertySetter;
+  IndexedPropertyQueryCallback indexedPropertyQuery;
+  IndexedPropertyDeleterCallback indexedPropertyDeleter;
+  IndexedPropertyEnumeratorCallback indexedPropertyEnumerator;
+  IndexedPropertyDefinerCallback indexedPropertyDefiner;
+  IndexedPropertyDescriptorCallback indexedPropertyDescriptor;
+  Persistent<Value> indexedPropertyInterceptorData;
+
+  SetterGetterInterceptor(SetterGetterInterceptor * setterGetterInterceptor):
+    namedPropertyGetter(setterGetterInterceptor->namedPropertyGetter),
+    namedPropertySetter(setterGetterInterceptor->namedPropertySetter),
+    namedPropertyQuery(setterGetterInterceptor->namedPropertyQuery),
+    namedPropertyDeleter(setterGetterInterceptor->namedPropertyDeleter),
+    namedPropertyEnumerator(setterGetterInterceptor->namedPropertyEnumerator),
+    namedPropertyDefiner(setterGetterInterceptor->namedPropertyDefiner),
+    namedPropertyDescriptor(setterGetterInterceptor->namedPropertyDescriptor),
+    namedPropertyInterceptorData(
+        nullptr, setterGetterInterceptor->namedPropertyInterceptorData),
+    indexedPropertyGetter(setterGetterInterceptor->indexedPropertyGetter),
+    indexedPropertySetter(setterGetterInterceptor->indexedPropertySetter),
+    indexedPropertyQuery(setterGetterInterceptor->indexedPropertyQuery),
+    indexedPropertyDeleter(setterGetterInterceptor->indexedPropertyDeleter),
+    indexedPropertyEnumerator(setterGetterInterceptor->indexedPropertyEnumerator),
+    indexedPropertyDefiner(setterGetterInterceptor->indexedPropertyDefiner),
+    indexedPropertyDescriptor(setterGetterInterceptor->indexedPropertyDescriptor),
+    indexedPropertyInterceptorData(nullptr, setterGetterInterceptor->indexedPropertyInterceptorData)
```

Comment:
Does this pass cpplint? It looks like its fairly long.

Reply:
done